packages/sessions-sdk-react/src/components/display-address.module.scss

```diff
@@ -0,0 +1,16 @@
+@use "../theme";
+
+.skeleton {
+  display: inline-block;
+  background: theme.color("skeleton");
+  height: 1em;
+  width: 6em;
```

Comment:
Minor nit but I think width should be set in terms of `theme.spacing`, not em units